examples/knative/powershell/kn-ps-email/test/send-cloudevent-test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo "Testing Function ..."
+curl -d@binary-payload -H "Content-Type: application/json" -H 'ce-specversion: 1.0' -H 'ce-id: id-123' -H 'ce-source: source-123' -H 'ce-type: binary' -H 'ce-subject: subject-123' -X POST localhost:8080
```

Comment:
IMHO better to use a real event type here 'ce-type: binary'

Reply:
Where is `binary-payload` defined? Should this be `test-payload.json` ?

Reply:
What should the example be?

Reply:
I was just wondering and assuming that `-d` takes an existing file (name) but could not find `binary-payload` anywhere in the commit?

Reply:
Yup, this was my bad.

This is fixed w/latest commit - https://github.com/vmware-samples/vcenter-event-broker-appliance/blob/005a4e3308c423855d14ef7a7d1fed8460aee848/examples/knative/powershell/kn-ps-email/test/send-cloudevent-test.sh#L4